Food_Tracker_App/app/src/main/java/com/example/foodtracker/ui/recipes/RecipesMainScreen.java

```diff
@@ -38,7 +41,20 @@ public class RecipesMainScreen extends AppCompatActivity implements
             addRecipe(receivedRecipe);
         }
     });
-
+    private final ActivityResultLauncher<Intent> recipeDisplayResultLauncher = registerForActivityResult(new ActivityResultContracts.StartActivityForResult(), activityResult -> {
+        if (activityResult.getData() != null && activityResult.getData().getExtras() != null) {
+            Recipe recipeToDelete = (Recipe) activityResult.getData().getSerializableExtra("DELETED_RECIPE");
+            deleteRecipe(recipeToDelete);
+        }
+    });
+/*
+    private final ActivityResultLauncher<Intent> editRecipeActivityResultLauncher = registerForActivityResult(new ActivityResultContracts.StartActivityForResult(), activityResult -> {
+        if (activityResult.getData() != null && activityResult.getData().getExtras() != null) {
+            Recipe recipeToEdit = (Recipe) activityResult.getData().getSerializableExtra("EDIT_RECIPE");
+            editRecipe(recipeToEdit);
+        }
+    } );
```

Comment:
We may need to have two result launchers... not too sure, we need a way of getting data from the edit to the display screen and then back to here directly after.